CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## Unreleased
 
-- ...
```

Comment:
This should remain under the `Unreleased` section and a new section with the version and date of release should be added instead

Reply:
That should be done when the release is actually made. Until Daniel is going to do that I am no adding a version number / date.

Reply:
Since it makes no sense to update the CHANGELOG twice just to add the release date, you should ask @HazAT the release date and then we will merge this PR just before releasing

Reply:
Jep, he can also commit to this branch ;) Just wanted to get the release notes done to keep it as quick as possible.